portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/search/searchportlet/Search.testcase

```diff
@@ -364,8 +364,8 @@ definition {
 		property app.server.types = "jboss,tcserver,tomcat,weblogic,websphere,wildfly";
 		property custom.properties = "search.container.page.default.delta=2";
 		property database.types = "db2,hypersonic,mariadb,mysql,oracle,postgresql,sqlserver,sybase";
-		property environment.acceptance = "true";
-		property portal.acceptance = "true";
+		property environment.acceptance = "quarantine";
```

Comment:
For some of our FI owned modules, our ci relevant query also run tests that have `property environment.acceptance = "true" as well. So I set this to quarantine. Hopefully that's all that's needed to properly quarantine. Rerunning relevant tests. @zhangyan0701 can you follow up with the test results if they're not finished before my end of day and `ci:forward` if looks good?